lib/ui/onboarding/edit_account/interactor/edit_account_event.dart

```diff
@@ -5,5 +5,5 @@ class EditAccountEvent with _$EditAccountEvent {
   const factory EditAccountEvent.initial() = _Initial;
   const factory EditAccountEvent.onNextPressed(InviteLinkData inviteLinkData) = _OnNextPressed;
   const factory EditAccountEvent.clearPageCommand() = _ClearPageCommand;
-  const factory EditAccountEvent.onAccountChange(String value) = _OnAccountChange;
+  const factory EditAccountEvent.onAccountChange(String value, Network network) = _OnAccountChange;

```

Comment:
@n13  this is not needed. You can get the network from the state inside the bloc, no need to pass it from the UI